lib/routes/users.js

```diff
@@ -64,6 +65,28 @@ router.post('/:id/towatch', async (request, response) => {
   }
 });
 
+router.post('/:id/towatch/multi', async (request, response) => {
```

Comment:
`/multi` is not RESTful.
Instead, you would use `/api/users/:userId/towatch` as route and check if you added multiple showIds or a single showed. 
See https://stackoverflow.com/questions/15370387/restful-api-endpoint-for-adding-removing-array-elements

Reply:
good point! thanks for the heads up. I changed the endpoint and refactored a bit